src/services/openletters/index.jsx

```
@@ -83,250 +96,383 @@ export const AddOpenLetter = ({
+      )}
+
+      {/* Letter Images */}
+      {letter?.images?.length > 0 && (
+        <div className="flex justify-center flex-wrap gap-2 mt-4 overflow-x-auto">
+          {letter.images.map((image, index) => (
+            <img
+              key={index}
+              src={image}
+              loading="lazy"
+              className="w-36 h-36 object-cover rounded-md shadow-md cursor-pointer"
+              alt={`Image ${index + 1}`}
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)